.github/workflows/release-deploy.yml

```diff
@@ -0,0 +1,52 @@
+# release-deploy.yml
+# releases and publish a version when a workflow_dispatch is triggered
+name: release-deploy
+
+on: 
+  workflow_dispatch:
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v3
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          cache: "yarn"
+
+      - name: Conventional Changelog Action
+        uses: TriPSs/conventional-changelog-action@v3
+        id: changelog
+        with:
+          github-token: ${{ secrets.github_token }}
+          skip-on-empty: 'false'
+          pre-commit: scripts/pre-commit.js
+
+      - name: Build
+        run: |
+          yarn install
+          yarn run build
+
+      - name: zip hypercrx.zip
+        run: zip release/hypercrx.zip -r build/*
+
+      - name: Release
+        uses: ncipollo/release-action@v1
+        with:
+          artifacts: "release/hypercrx.*"
+          token: ${{ secrets.GITHUB_TOKEN }}
+          tag: ${{ steps.changelog.outputs.tag }}
+          body: ${{ steps.changelog.outputs.clean_changelog }}
+          git-message: 'chore(release): {version}'
```

Comment:
Haha, wrong place to put 'git-message'

<img width="1366" alt="image" src="https://user-images.githubusercontent.com/32434520/174967458-4a6c9784-e816-4261-8bd3-5826cc21ca52.png">